src/types/motions.ts

```diff
@@ -84,3 +85,12 @@ export interface MotionQuery {
   motionData: MotionData;
   createdAt: string;
 }
+
+export interface Event {
```

Comment:
@rdig Related to Juliana's question about combining events and system messages, if it's fine to have one model to capture both, I propose we rename it to `Messages`.

"Event" is ambiguous since we'd not be tracking network events exclusively, and even in that event, since we're only recording a small-subset of the actual event's data, calling it an event feels misleading. 

Really what we're capturing is the data needed to display a message to the user in the motion's page, so I feel this should drive the naming of the model.

Reply:
System Messages are generated locally based on various states of events. I don't have clear idea how to handle them now. 

From a separation concern, I'd want a separate model, from a logic reuse concern I'd want to store them in the same model. Maybe have a call about this?

At any rate, do note that System Message won't have the same values as a normal event does.

Reply:
Is there another reason, aside from to display messages, to track the "events" on the motion data model? If not, I think we just call it `messages` or `systemMessages`. I.e. what would be the reason to keep them separate? 

I say "events" because we're not actually storing the full event object, we're storing a subset of its fields that are useful for displaying messages. So really it's the same concern, as far as I can tell.